linphone.xcodeproj/project.pbxproj

```diff
@@ -1881,25 +1886,23 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				225F6C091B380E1B00A570DC /* libilbc.a in Frameworks */,
+				50F024BA1B910C6C000BC09F /* libxml2.2.dylib in Frameworks */,
 				225F6C081B380E0000A570DC /* libSKP_SILK_SDK.a in Frameworks */,
-				225F6C011B380DE600A570DC /* libxml2.a in Frameworks */,
 				C6F476531B27157D00F4C2A7 /* CoreLocation.framework in Frameworks */,
-				F0B026F31AA710AF00FF49F7 /* libiconv.dylib in Frameworks */,
```

Comment:
Since I removed libiintl support from liblinphone. There was no harm in removing a library that is not used (linked) right now but my priority was to get it compiling/linking correctly so I can get the entire branch in good shape but anyways i agree with removing dependencies libraries is questionable at best. (I was getting pretty frustrated at the time when it kept failing at linker stage),